src/Client.php

```diff
@@ -305,7 +304,7 @@ public function listDatabaseNames(array $options = []): Iterator
      * @throws InvalidArgumentException for parameter/option parsing errors
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function listDatabases(array $options = []): DatabaseInfoIterator
+    public function listDatabases(array $options = []): Iterator
```

Comment:
Noted that these methods already have a templated return type, as do the Operation class `execute()` methods.